yamale/validators/tests/test_validate.py

```diff
@@ -22,6 +38,135 @@ def test_string():
     assert not v.is_valid(1)
 
 
+def test_validator_pathname():
+    '''
+    Unit test the `Pathname` validator.
+    '''
+
+    v = val.Pathname()
+
+    # Directory guaranteed to exist. See Pathname._is_valid().
+    root_dirname = os.environ.get('HOMEDRIVE', 'C:') \
```

Comment:
It stems around the fact that an invisible character can break the code. Now, at first this seems silly since Python is a whitespace significant language, but this whitespace happens to be at the end. Maybe fragile is a strong word, but I know I'd be confused if I accidentally added a space and code started breaking with no visible change.